core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -139,6 +139,10 @@ CFG_MMAP_REGIONS ?= 23
 CFG_DTB_MAX_SIZE ?= (256 * 1024)
 CFG_CORE_ASLR ?= n
 
+
+# Embed public part of this key in OP-TEE OS
+RPROC_SIGN_KEY ?= keys/default.pem
```

Comment:
I think `RPROC_SING_KEY` should be introduced later, with the patch introducing the RPROC PTA, where it is used, hence outside of this P-R.

Reply:
I think copying the python script into the devkit makes sense in this series.

Reply:
done